salt/repositories/containers.repo

```diff
@@ -4,4 +4,9 @@ type=rpm-md
 gpgcheck=0
 enabled=1
 autorefresh=1
-baseurl=http://download.opensuse.org/repositories/Virtualization:/containers/openSUSE_Leap_{{ grains['osrelease'] }}/
\ No newline at end of file
+{% if grains['oscodename'].startswith("openSUSE Leap") %}
+baseurl=http://download.opensuse.org/repositories/Virtualization:/containers/openSUSE_Leap_{{ grains['osrelease'] }}/
+{% else %}
+# not really important: it will not be used
```

Comment:
Yeah, we could do that... Anyway, I think we should add a case for Tumbleweed at some moment, just in case someone want to try it.